quasarpy/cli/__init__.py

```diff
@@ -68,51 +67,52 @@ def detect(path, format, solution, create_issue, output, offline) -> None:
     issue_handler = IssueHandler(repo=Repository()) if create_issue else None
     redis_config = RedisConfig()
 
-    if solution:
+    if not solution:
         raise NotImplementedError("Solution flag not implemented yet")
```

Comment:
This error should be raised when solution flag is used, which has default value as True so, `not` will make it run every time solution flag is not used
```suggestion
if solution:
        raise NotImplementedError("Solution flag not implemented yet")
```